test/parallel/test-util-types.js

```diff
@@ -291,3 +292,7 @@ for (const [ value, _method ] of [
   assert.ok(!types.isCryptoKey());
   assert.ok(!types.isKeyObject());
 }
+
+builtinModules.forEach((module) => {
+  assert.ok(types.isBuiltinModule(module));
+});
```

Comment:
How about also testing some values that are definitely not built-in module names (valid inputs or otherwise)?